docusaurus.config.js

```diff
@@ -48,124 +48,125 @@ module.exports = {
             },
             {
               label: 'Magento 2',
-              to: '/docs/platform/integration/magento2/overview',
+              to: '/docs/platform/integration/magento2/overview'
             },
             {
               label: 'Stripe',
-              to: '/docs/platform/integration/stripe',
+              to: '/docs/platform/integration/stripe'
             },
             {
               label: 'Mollie',
-              to: '/docs/platform/integration/mollie',
+              to: '/docs/platform/integration/mollie'
             },
             {
               label: 'Wordpress',
-              to: '/docs/platform/integration/wordpress',
-            },
-          ],
+              to: '/docs/platform/integration/wordpress'
+            }
+          ]
         },
         {
           title: 'Community & Help',
           items: [
             {
               label: 'Community Chat',
-              href: 'http://slack.deity.io/',
+              href: 'http://slack.deity.io/'
             },
             {
               label: 'Twitter',
-              href: 'https://twitter.com/deity_commerce',
+              href: 'https://twitter.com/deity_commerce'
             },
             {
               label: 'Instagram',
-              href: 'https://instagram.com/deity_commerce',
+              href: 'https://instagram.com/deity_commerce'
             },
             {
               label: 'YouTube',
-              href: 'https://www.youtube.com/channel/UCCyszDV63yrqFHUY1uWf4mQ',
+              href: 'https://www.youtube.com/channel/UCCyszDV63yrqFHUY1uWf4mQ'
             },
             {
               label: 'Contact Us',
-              href: 'https://deity.io/contact',
-            },
-          ],
+              href: 'https://deity.io/contact'
+            }
+          ]
         },
         {
           title: 'Useful Links',
           items: [
             {
               label: 'Marketing Site',
-              href: 'https://deity.io/',
+              href: 'https://deity.io/'
             },
             {
               label: 'Status',
-              href: 'https://status.deity.io/',
+              href: 'https://status.deity.io/'
             },
             {
               label: 'Blog',
-              href: 'https://medium.com/deity-io',
+              href: 'https://medium.com/deity-io'
             },
             {
               label: 'Privacy Policy',
-              href: 'https://deity.io/privacy-policy',
-            },
-          ],
-        },
+              href: 'https://deity.io/privacy-policy'
+            }
+          ]
+        }
       ],
       logo: {
         alt: 'DEITY Logo',
-        src: 'img/deity-logo.svg',
+        src: 'img/deity-logo.svg'
       },
-      copyright: `Copyright © ${new Date().getFullYear()} DEITY B.V.`,
+      copyright: `Copyright © ${new Date().getFullYear()} DEITY B.V.`
     },
     image: 'img/deity-logo.svg',
 
     algolia: {
+      appId: '3SVVV93FRW',
```

Comment:
@hobadams do you know whose apiKey is this? (on line 153)

I put appId from another account, but I doubt it will work properly

Reply:
@Borales - It's come from doc search, i don't think it's the same as a regular Algoilo api key

Reply:
applied for a new one, I'll wait